### PR TITLE
Class signedness

### PR DESCRIPTION
**Pull request by vasilissoti**

### CODE REVIEW

**Comment by DaveMcEwan** [comment]
On a class-datatype (struct/enum) port, does the signedness come from the struct/enum definition?

Does it make sense to add a new member to SvPort like `pub class_identifier: Option<String>`? Then some future work can analyse struct/enum/class definitions, so a reader could lookup the class definition by the identifier to find all the port info.

**Comment by vasilissoti** [comment]
Up to know once datatype = class since no signedness was found in Ansiportdeclaration, it was set by default to unsigned which I believe does not apply in our case.

Yes on the upcoming plans I have also included adding an SvClass? struct that will contain any struct/enum found overall from the parsing and prior  to returning the SvData if any match that exists between a class_identifier in SvPort and the identifier found in SvClass will lead to the data of SvClass being transferred in SvPort.

**Comment by DaveMcEwan** [comment]
> that will lead to the data of SvClass being transferred in SvPort

I reckon that transferring all of the class information to SvPort will be a bit too complex, especially when namespaces are used. Probably simpler to just add the identifier to SvPort, and have a list of enums in something like `SvModuleDeclaration.enums` (similar for classes/structs).

**Comment by vasilissoti** [comment]
Of course, I'm merging this branch with the latest version of upstream/master and then I will add any necessary implementation and update the PR.

**Comment by vasilissoti** [comment]
Fixed.

**Comment by vasilissoti** [comment]
Fixing the conflict.

File: src/sv_port.rs [hunk 1]
```diff
@@ -197,12 +197,24 @@ fn port_nettype_ansi(
...
-    match signedness {
-        Some(RefNode::Signing(sv_parser::Signing::Signed(_))) => SvSignedness::Signed,
-        Some(RefNode::Signing(sv_parser::Signing::Unsigned(_))) => SvSignedness::Unsigned,
-        _ => SvSignedness::Unsigned,
+fn port_signedness_ansi(
+    m: &sv_parser::AnsiPortDeclaration,
+    datatype: &SvDataType,
+) -> Option<SvSignedness> {
+    match datatype {
+        SvDataType::Class => return None,
```
Comment (DaveMcEwan) [inline src/sv_port.rs:205]: Maybe I'm missing something, but I don't see the need for `return` in this function.

Comment (vasilissoti) [reply src/sv_port.rs]: Just removed it.

**Pull request merged**